.buildkite/pipelines/pull-request/build-benchmark.yml

```diff
@@ -1,6 +1,6 @@
 config:
   allow-labels: build-benchmark
-  trigger-phrase: '.*run\W+elasticsearch-ci/build-bench.*'
+  trigger-phrase: '.*run\W+jenkins\W+elasticsearch-ci/build-bench.*'
```

Comment:
This is a buildkite pipeline. Should we be changing the trigger phrase here?

Reply:
No, thanks for catching. Find-and-replace got me